example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
+
+interface UseGravityArgs {
+  moverMass: number;
+  attractorMass: number;
+  attractor: Vector<number>;
+}
+
+const applyForceForStep = (
+  config: UseGravityArgs,
+  mover: Entity,
+  entity: Entity
+) => {
+  const force = forceV({
+    ...config,
+    mover: mover.position
+  });
+
+  return applyForce({
+    force,
+    moverMass: config.moverMass,
+    acceleration: mover.acceleration,
+    velocity: mover.velocity,
+    position: mover.position
+  });
+};
+
+const run = (config: UseGravityArgs) => {
+  const { start, stop } = rAF();
+
+  start((timestamp, lastFrame, mover) => {
+    const diffTime = timestamp > lastFrame + 64 ? 0 : lastFrame;
+    const steps = Math.floor(timestamp - diffTime);
+
+    let entity: Entity;
+
+    if (steps > 0) {
+      for (let i = 0; i < steps; i++) {
+        entity = applyForceForStep(config, mover, entity);
+      }
+    } else {
+      entity = applyForceForStep(config, mover, entity);
+    }
+
+    return entity;
+  });
+
+  setTimeout(() => {
+    stop();
+  }, 1000);
+};
+
+run({
+  moverMass: 20,
+  attractorMass: 10000,
+  attractor: [500, 500]
+});
```

Comment:
This sort of configuration will be a combination of user-provided (part of the API) or derived from user input (i.e. the location of `attractor` will actually be determined by placing `mover` at (0, 0) and placing `attractor` at (r, 0) for 1d gravity. For `useGravity2d` we'll ask them to provide a 2d location for their `attractor`.